packages/components-native/src/Typography/Typography.style.ts

```diff
@@ -58,6 +43,31 @@ export const typographyTokens: { [index: string]: TextStyle } = {
   displayRegularBlack: {
     fontFamily: "jobberpro-blk",
   },
+};
+
+const fonts = Platform.select({
+  web: webFonts,
+  default: deviceFonts,
+});
+
+/**
+ * `StyleSheet` for Typography.tsx.
+ *
+ * If you find yourself needing to use what's inside this object on files other
+ * than `<Typography />`, please import from `@jobber/components-native` instead.
+ *
+ * ```
+ * import { typographyStyles } from "@jobber/components-native"
+ * ```
+ */
```

Comment:
unrelated but i see it now, this comment is talking about the line 373 https://github.com/GetJobber/atlantis/blob/21624543aa08c13320e3b1e2731a9549e243f16f/packages/components-native/src/Typography/Typography.style.ts#L373

Reply:
Nice catch, I changed it to be above the `typographyStyles` const.